lib/Math.db4

```diff
@@ -8,7 +8,7 @@ fn is_neg(a) {
 
 fn abs(a) {
     if (a < 0) {
-        return -a;
+        return a * -1;
```

Comment:
![no negative numbers](https://github.com/EdenComp/GLaDOS/assets/54242096/804cfed6-1c74-41ec-b107-a39ad081bea5)

Reply:
Its on you buddy

Reply:
mais je valide le meme